blogposts/2021/show-us-your-calendar.md

```diff
@@ -0,0 +1,166 @@
+
+The nonlinear workday, [according to GitLab](https://about.gitlab.com/company/culture/all-remote/non-linear-workday/), the company that popularized the term, “decouples time from work and acts as a forcing function to embrace asynchronous workflows.”
+
+![Nonlinear calendar graphic](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Nonlinear.png)
+
+For Jean, this means using the morning to take care of the chores that most would do after work, such as grocery shopping and walking the dog. From the late morning to the afternoon, he focuses on meetings with his teammates and completing work leftover from the previous day.
+
+> “I’ve found myself playing video games in the afternoon and I’ve found myself meeting up with friends.” - Jean du Plessis, director of engineering
+
+In the afternoon, Jean is frank: he might just nap. After, around 3 pm (his time), is when most people in the east coast of the United States start to wake up, so the next three hours have some of the best overlap for meetings. From 6:00 pm to 8:30 pm is family time: “We have supper together, kids get bathed, we read a story.” And from 8:30 pm to 10:00 pm, he handles any stray follow-up meetings and completes some light work.
+
+![Slack message from Jean regarding his schedule](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Screen%20Shot%202021-09-13%20at%201.20.11%20PM.png)
+
+<p class="font-italic text-center">
+Jean's message to the team about his work schedule.
+</p>
+
+For Jean, the nonlinear work day isn’t merely a traditional workday, fragmented: “I’m picky about what I do in the evenings.” He tends to schedule meetings that he doesn’t have to lead during his evening sessions–meetings where he only has to listen or provide input. He optimizes his entire day based on what he does best when.
+
+And that includes rest: “Friday is my recovery day,” he says. “If I felt like I put in a lot of effort on the days leading up to Friday, I’ll take Friday easy. I’ve found myself playing video games in the afternoon and I’ve found myself meeting up with friends.”
+
+One benefit of the traditional 9 to 5 is that some sense of work/life balance is built into its limited 40 hours. Even still, many people find themselves burnt out. Does a nonlinear day threaten to make this even worse, given the potential for even more hours worked?
+
+Not for Jean. “For me,” he says, “maintaining a healthy work/life balance isn’t about the time. It’s about how I feel: does my work let me focus on my personal life when I need to? Does my personal life allow me to focus on my work when I need to? Are the stakeholders in my life, my personal and my work life, feeling like their needs are met?”
+
+If the answers to these questions are “yes,” “yes,” and “yes,” then Jean feels balanced. And for him, a nonlinear workday makes it easier to keep answering “yes.”
```

Comment:
@jeanduplessis The calendar post you contributed to is almost ready to go live. Would you mind taking a quick glance at your section and letting me know if it's accurate?